src/Defaults/MergeTag/Taxonomy/TermID.php

```diff
@@ -17,22 +17,32 @@
  */
 class TermID extends IntegerTag {
 
+	/**
+	 * Property name
+	 *
+	 * @var string
+	 */
+	protected $property_name;
+
 	/**
 	 * Merge tag constructor
 	 *
 	 * @since 5.2.2
+	 * @param array $params merge tag configuration params.
 	 */
-	public function __construct() {
+	public function __construct( $params = [] ) {
+
+		$this->set_property_name( $params, 'property_name', 'term' );
 
 		$args = wp_parse_args(
 			[
-				'slug'        => 'term_ID',
```

Comment:
Please don't update the slug for those, it will break all the notifications already set by the users.

Reply:
I'm wondering if this change will break it after all, can you check if all the triggers use the `term` as the property name?